src/test/ui/pattern/issue-68394-let-pat-runtime-value.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0080]: runtime values cannot be referenced in patterns
```

Comment:
I would like the wording of this error to be tweaked a bit.

Reply:
What's wrong with the current?

Reply:
I feel it is a bit jargony, but I guess it's no worse than other more common errors we emit.